CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ If you run into any problems we have plenty of people who are willing to help; s
 
 The [Getting Started Quick Start Guide](contributing/getting-started-quick.md) has installation steps and instructions for building/testing AMP.
 
-[DEVELOPING.md](DEVELOPING.md) has some more advanced instructions that may be necessary depending on the complexity of the changes you are making.
+[DEVELOPING.md](contributing/DEVELOPING.md) has some more advanced instructions that may be necessary depending on the complexity of the changes you are making.
```

Comment:
Is there any chance you could move these changes into a single PR so they can be committed atomically?